MetaMask/metamask-extension
# Migrate LavaMoat validation to GitHub Actions

## Explanation

The LavaMoat policy and allow-scripts config validation has been moved from CircleCI to GitHub Actions. It should run in the same conditions, which is on `master` and on release candidate branches.


The LavaMoat policies are validated in parallel, one job per policy.

## Manual Testing Steps

See that the validation is skipped on this branch.

In a previous run the condition was comment out so that we could validate that the policy generation worked.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
